libs/environment/src/hooks/use-environment.ts

```diff
@@ -159,7 +163,20 @@ export const useInitializeEnv = () => {
 const fetchConfig = async (url?: string) => {
   if (!url) return [];
   const res = await fetch(url);
```

Comment:
if github is down or similar we need a way of safely failing in a reasonable way. We don't want to be susceptible to crashes from the downtime of an application we have no control over.